Index.html

```diff
@@ -217,6 +226,7 @@ <h2 class="hh2">Multi-Post Story</h2>
                     <input type="email" name="Email" required placeholder="Email" id="email">
                     <textarea maxlength="500" name="message" required placeholder="Enter You Message Here"></textarea>
                     <button type="submit" class="mmb">See Project</button>
+                    <button type="submit" name="send" class="dsk">Send</button>
```

Comment:
- [ Optional  ] For desktop version of your contact section you missed "Let chat"  heading , Kindly Fix it.

Your contact section look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200140463-578ba74f-5a69-45a0-bca9-933c7689a47d.PNG)

The Figma Contact section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200140492-13cb068c-e487-4898-ad4b-229eb53eaf58.PNG)